tensorboard/webapp/widgets/card_fob/card_fob_types.ts

```diff
@@ -25,31 +25,31 @@ export interface TimeSelection {
  * Only used for internal analytics.
  */
 export enum TimeSelectionAffordance {
-  NONE,
+  NONE = 'NO_AFFORDANCE',
   // Dragging the extended line above a fob.
-  EXTENDED_LINE,
+  EXTENDED_LINE = 'EXTENDED_LINE',
   // Dragging the fob.
-  FOB,
+  FOB = 'FOB',
   // Clicking the deselect button.
-  FOB_REMOVED,
+  FOB_REMOVED = 'FOB_REMOVED',
   // Typing the step in fob.
-  FOB_TEXT,
+  FOB_TEXT = 'FOB_TEXT',
   // Typing the step in setting pane.
-  SETTINGS_TEXT,
+  SETTINGS_TEXT = 'SETTINGS_TEXT',
   // Dragging the slider in setting pane.
-  SETTINGS_SLIDER,
+  SETTINGS_SLIDER = 'SETTINGS_SLIDER',
```

Comment:
How about making the string lower case and separated with space? i.e. "settings slider" Then probably the extra conversion could be saved. What do you think?

Reply:
It mostly could except for the situation with `NONE`. I'll go ahead and change them all though.

Reply:
I'm going to change to use camel case as that is what they are converted to internally.

Reply:
nice. I misremembered the converted string type for internal analysis. lgtm